tools/code_format/check_format.py

```diff
@@ -252,7 +252,7 @@ def check_bazel_tool(name, path, var):
                     "{} environment variable to specify the path. Example:\n"
                     "    export {}=`which {}`\n"
                     "If you don't have {} installed, you can install it by:\n"
-                    "    go get -u github.com/bazelbuild/buildtools/{}".format(
+                    "    go install github.com/bazelbuild/buildtools/{}@latest".format(
```

Comment:
can we tie this to a specific version to ensure reproducibility - im also wondering about whether we can put this in a go file that dependabot can monitor

Reply:
hmm just realizing this is install advice rather than the install

i forget the current state of this script but ideally we just handle this all with bazel

Reply:
@phlax 
It seems there is no specific version for buildifier and others: https://github.com/bazelbuild/buildtools/tree/master/buildifier, especially when installing via go.

Reply:
yeah - we used to have a bazel recipe for these build tools (in `bazel/repository_locations.bzl` etc) - i removed them because they werent being used at the time - but we could readd and make this script work with them

Reply:
for ref https://github.com/envoyproxy/envoy/pull/20182

Reply:
checking the ci code ... 

https://github.com/envoyproxy/envoy/blob/e0bca625f98a0270a78e7c1aa787ca4241eb7d60/tools/code_format/paths.py#L6-L15

it seems ci is not so ideal either, essentially falling back to system tools after some non-deterministic go discovery attempt